packages/@aws-cdk/aws-appconfig-alpha/lib/environment.ts

```diff
@@ -348,8 +348,9 @@ export abstract class Monitor {
    * @param monitorsProperty The monitors property.
    */
   public static fromCfnMonitorsProperty(monitorsProperty: CfnEnvironment.MonitorsProperty): Monitor {
+    if (monitorsProperty.alarmArn === undefined) { throw new Error('need this prop'); }
```

Comment:
sorry! we need a better error message than this. This was a placeholder :)

Reply:
Oops! Updated